### PR TITLE
Test or CASSANDRA-11891

### PR DESCRIPTION
**Pull request by thobbs**

### CODE REVIEW

File: commitlog_test.py [hunk 1]
```diff
@@ -160,6 +161,77 @@ def _provoke_commitlog_failure(self):
...
+            if replay_complete:
+                break
+        else:
+            self.fail("Did not finish commitlog replay within 120 seconds")
+
+        debug("Reconnecting to node")
+        session = self.patient_cql_connection(node1)
+        debug("Make query to ensure data is present")
+        res = list(session.execute("SELECT * FROM Test.mytable"))
+        self.assertEqual(num_rows, len(res))
```
Comment (ptnapoleon) [inline commitlog_test.py]: Can you add `res` as the third arg to this?

File: commitlog_test.py [hunk 2]
```diff
@@ -160,6 +161,77 @@ def _provoke_commitlog_failure(self):
...
+        for i in xrange(num_rows):
+            session.execute("INSERT INTO Test.mytable (a, b, c) VALUES (0, %d, %d)" % (i, i))
+
+        node1.stop(gently=False)
+        node1.mark_log_for_errors()
+
+        debug("Verify commitlog was written before abrupt stop")
+        commitlog_dir = os.path.join(node1.get_path(), 'commitlogs')
+        commitlog_files = os.listdir(commitlog_dir)
+        self.assertTrue(len(commitlog_files) > 0)
```
Comment (ptnapoleon) [inline commitlog_test.py]: Can you add `commitlog_files` as the second arg to this method?

**Comment by ptnapoleon** [comment]
+1 after minor nits to improve debug output on assertion failure.

File: commitlog_test.py (hunk 2 above)
Comment (mambocab) [inline commitlog_test.py]: It'd be better to use `self.assertNotEqual(commitlog_files, [])`.

**Comment by thobbs** [comment]
Looks like the flake8 failure is in an unrelated file, and the previous commits had no problems, so I'm going to merge this.  Thanks for the reviews.

**Pull request merged**